### PR TITLE
LPS-198224 Create test for DDMStructureLinkTableReferenceDefinition

### PR DESCRIPTION
**Pull request by gislayne-vitorino**
This is an update for: [LPS-198224](https://liferay.atlassian.net/browse/LPS-198224)

### CODE REVIEW

**Comment by gislayne-vitorino** [comment]
ci:test:publications

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/change/tracking/test/DDMStructureLinkTableReferenceDefinitionTest.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	@Rule
+	public static final AggregateTestRule aggregateTestRule =
+		new AggregateTestRule(
+			new LiferayIntegrationTestRule(),
+			PermissionCheckerMethodTestRule.INSTANCE);
+
+	@Override
+	protected CTModel<?> addCTModel() throws Exception {
+		return _ddmStructureLinkLocalService.addStructureLink(
+			_portal.getClassNameId(JournalArticle.class),
```
Comment (pei-jung) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/change/tracking/test/DDMStructureLinkTableReferenceDefinitionTest.java:41]: Use `ClassNameLocalService` to get the classNameId

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/change/tracking/test/DDMStructureLinkTableReferenceDefinitionTest.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+			RandomTestUtil.randomLong(), RandomTestUtil.randomLong());
```
Comment (pei-jung) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/change/tracking/test/DDMStructureLinkTableReferenceDefinitionTest.java:42]: We need a DDMStructure object to create the link.

**Comment by gislayne-vitorino** [comment]
ci:test:sf

**Comment by gislayne-vitorino** [comment]
ci:test:publications

**Comment by pei-jung** [comment]
ci:forward:force

**Pull request closed without merging**